### ISSUE TITLE
Whatsapp crashes when linking to whatsapp web.

### ISSUE THREAD

**Issue by mukulkadel** [mukulkadel/mwatoi#7]

**Comment by dtiziano** [comment]
having the same problem. even after reinstalling WhatsApp and restoring the data directly from the the iCloud backup.

**Comment by Saitu** [comment]
Yes, still happening as of latest version of mwatoi. Reinstalling WhatsApp and restoring from icloud backup (which was made after running mwatoi and includes the old chats) causes whatsapp to crash. Whatsapp Web is stuck at "organizing messages" and then brings me back to QR code screen throwing an error which says "Your device could not link due to a sync issue. Please relink your device."

**Comment by fahadismail95** [comment]
> Yes, still happening as of latest version of mwatoi. Reinstalling WhatsApp and restoring from icloud backup (which was made after running mwatoi and includes the old chats) causes whatsapp to crash. Whatsapp Web is stuck at "organizing messages" and then brings me back to QR code screen throwing an error which says "Your device could not link due to a sync issue. Please relink your device."

did you get any solution to this ?

**Comment by mukulkadel** [comment]
Issue is still there. I haven't been able to find the root cause yet.

**Comment by ashishbansal6** [comment]
Did anyone got solution to this problem?Whatsapp is crashing on iphone when trying to log in to whatsapp web.

**Comment by mukulkadel** [comment]
Hi, with the recent WhatsApp update and web app changes, I'm not facing this issue.

**Comment by ashishbansal6** [comment]
> Hi, with the recent WhatsApp update and web app changes, I'm not facing this issue.

I tried with latest version WhatsApp ,app is crashing in iPhone when trying to connect to

**Comment by gigara** [comment]
I tried with version 2.22.20.75, still crashes.

**Comment by micalevy108** [comment]
Hi guys, I have exactly the same problem, after syncing whatsapp web, my whatsapp on iOS crashes, any solution?

**Comment by thiagograzzi** [comment]
Hi Guys... i have the same problem here, also i was in contact with whatsapp support but there is no solution.
Any of you got it fixed anyway?

**Comment by micalevy108** [comment]
No, been trying to fix it last 8 months. No reply from qhatsappMaybe we have to delete all conversatiosn?On 30 May 2023, at 15:58, thiagograzzi ***@***.***> wrote:﻿
Hi Guys... i have the same problem here, also i was in contact with whatsapp support but there is no solution.
Any of you got it fixed anyway?
